server/src/event/event.service.ts

```diff
@@ -224,6 +224,15 @@ export class EventService {
     });
   }
 
+  async getRecommendedEventsForUser(user: User, lat: number, long: number) {
```

Comment:
Maybe add the number of recommendations as a parameter for this function instead of it just being 4.

Reply:
I agree